hooks/tk-3dsmaxplus_actions.py

```diff
@@ -79,7 +79,39 @@ def generate_actions(self, sg_publish_data, actions, ui_area):
                                       "description": "This will insert a reference to this file into the current scene."} )        
         
         return action_instances
-                
+
+    def execute_action_on_selection(self, name, action_params):
```

Comment:
True! I'll go for `execute_multiple_actions`, which is generic enough and less error prone than `execute_actions`.